packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1046,7 +1031,7 @@ new PrismaClient({
           if (nextMiddleware) {
             // we pass the modified params down to the next one, & repeat
             // calling `next` calls the consumer again with the new params
-            return runInChildSpan(spanOptions.middleware, async (span) => {
+            return runInChildSpan(spanOptions.middleware, (span) => {
```

Comment:
unrelated, safe change?

Reply:
Yep, save to remove (pointed out by eslint)